Source/CNTKv2LibraryDll/API/CNTKLibrary.h

```diff
@@ -4142,27 +4142,29 @@ namespace CNTK
     }
 
     ///
-    /// Convolution 
+    /// Convolution
     ///
     CNTK_API FunctionPtr Convolution(const Variable& convolutionMap,
-                                     const Variable& operand, 
+                                     const Variable& operand,
                                      const NDShape& strides = { 1 },
                                      const std::vector<bool>& sharing = { true },
                                      const std::vector<bool>& autoPadding = { true },
-                                     size_t maxTempMemSizeInSamples = 0, 
+                                     const NDShape& dilation = { 1 },
+                                     size_t maxTempMemSizeInSamples = 0,
                                      const std::wstring& name = L"");
 
     ///
     /// Convolution transpose
     ///
     CNTK_API FunctionPtr ConvolutionTranspose(const Variable& convolutionMap,
-        const Variable& operand,
-        const NDShape& strides = { 1 },
-        const std::vector<bool>& sharing = { true },
-        const std::vector<bool>& autoPadding = { true },
-        const NDShape& outputShape = { 0 },
-        size_t maxTempMemSizeInSamples = 0,
-        const std::wstring& name = L"");
+                                              const Variable& operand,
+                                              const NDShape& strides = { 1 },
+                                              const std::vector<bool>& sharing = { true },
+                                              const std::vector<bool>& autoPadding = { true },
+                                              const NDShape& dilation = { 1 },
```

Comment:
We can't run full EndToEnd test on our side due to data dependency. There might be issue I did not catch.